include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -898,18 +841,36 @@ namespace detail
         }
     };
 
+    namespace bool_repr
```

Comment:
This is for now a duplication of the same code inside `AttributeTypes< bool >`. The latter will be removed as soon as the new ADIOS2 schema is fully removed.